auth/src/main/java/com/firebase/ui/auth/provider/FacebookProvider.java

```diff
@@ -47,26 +46,27 @@
     protected static final String ERROR_MSG = "err_msg";
 
     private static final String TAG = "FacebookProvider";
-    private static final String APPLICATION_ID = "application_id";
     private static final String EMAIL = "email";
     private static final String PUBLIC_PROFILE = "public_profile";
 
+    private final List<String> mScopes;
     private CallbackManager mCallbackManager;
     private IDPCallback mCallbackObject;
 
-    public FacebookProvider (Context appContext, IDPProviderParcel facebookParcel) {
+    public FacebookProvider (
+            Context appContext,
+            List<String> scopes) {
```

Comment:
Why not just pass through the IDPConfig here?  That way you only have to change this once.